command-expansion-server/src/app.ts

```diff
@@ -433,6 +433,44 @@ export interface SeqBuilder {
   ): Sequence;
 }
 
+//generate a sequence JSON from a sequence file
+app.post('/get-sequence-edsl', async (req, res, next) => {
```

Comment:
I think the name of this endpoint is a little confusing. I would have it reference that we are going from edsl -> seqjson

Reply:
I couldn't really think of a name but here is my second attempt:

`get-seqjson-for-sequence-standalone`